src/views/statutory/PositionsList.vue

```diff
@@ -228,7 +228,10 @@ export default {
     },
     // TODO: This is a temporary solution, in the future, we want to display either the "first" deadline OR the force_end deadline
     displayFinalDeadline () {
-      return moment().isAfter(this.positions[0].ends)
+      if (this.positions.length > 0) {
```

Comment:
What if this.positions does not exist?

Reply:
It can't be null, since it is initialized as an empty array at line 195